test/verify/check-packagekit

```diff
@@ -117,7 +117,17 @@ class TestUpdates(MachineCase):
         # Cancel button should eventually get disabled
         b.wait_present(".content-header-extra td button:disabled")
 
-        # should have succeeded and show restart page; cancel
+        # update log is not visible by default
```

Comment:
not visible and not present can have different meaning in these tests - perhaps the comment should be clearer?